doc/nrf/gs_modifying.rst

```diff
@@ -213,14 +223,19 @@ Changes are picked up immediately, and you do not need to re-open the project in
 While it is possible to edit the :file:`.config` file directly, you should use SES or a tool like menuconfig or guiconfig to update it.
 These tools present all available options and allow you to select the ones that you need.
 
+To edit the file in |VSC|, use the nRF Kconfig extension.
+This extension organizes the Kconfig option in a hierarchical list and lets you select and deselect the desired options.
+To save the changes made using the nRF Kconfig extension, click the :guilabel:`Save` button.
+Read the `nRF Kconfig`_ documentation for more information.
```

Comment:
This whole paragraph should be merged with the previous one about menuconfig, I think. This is describing "menuconfig", but for VS Code. The nRF Kconfig extension does a lot more than this Kconfig menu interface, so I think we should be referring to this feature as the VS Code Kconfig menu (or whatever we ended up calling it, @Avalei )

Reply:
In a conversation on May 9th we decided that we will use "_nRF Kconfig when referring to the extension as a whole, and "the nRF Kconfig GUI" when talking about the interface itself._"